c++/src/Compression.cc

```diff
@@ -463,6 +469,7 @@ DIAGNOSTIC_PUSH
   }
 
   bool DecompressionStream::Next(const void** data, int*size) {
+    DEFINE_AUTO_STOPWATCH(metrics, DecompLatencyUs, DecompCount);
```

Comment:
The stop watch should not be put here since it includes the io latency. If you move this line to line 498 right before the definition of **size_t availableSize**, what we got is the pure decompression time.

Reply:
The NextDecompress function will call the readBuffer function which includes the io operation. Currently the pure decompression time is got by subtracting the time taken by the readBuffer function.